apps/www/lib/translations.json

```diff
@@ -4378,15 +4426,15 @@
     },
     {
       "key": "questionnaire/submit",
-      "value": "abschicken"
+      "value": "Abschicken"
     },
     {
       "key": "questionnaire/publish",
-      "value": "publizieren"
+      "value": "Publizieren"
     },
     {
       "key": "questionnaire/update",
-      "value": "aktualisieren"
+      "value": "Aktualisieren"
     },
```

Comment:
Why are these written in lowercase now?

Reply:
in uppercase. Other strings like Später or Weiter also start with Uppercase, so I thought it is more consistent